### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by David-Bevan** [Mattk70/Chirpity-Electron#117] [General]
**Describe the bug**
trying to download chirpy onto my Mac to assist my Nocmig analysis. please find attached screen grabs.  
<img width="1440" alt="Screenshot 2024-04-03 at 22 43 13" src="https://github.com/Mattk70/Chirpity-Electron/assets/165961634/9019de48-92f6-4cab-a2d6-3882835bee36">
<img width="1440" alt="Screenshot 2024-04-03 at 22 43 32" src="https://github.com/Mattk70/Chirpity-Electron/assets/165961634/8b5d00c5-3eba-45a0-b841-4199b50de70c">

**To Reproduce**

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem. The Help > Diagnostics window will often be useful - as it captures important system and settings information.

**Additional context**
Add any other context about the problem here. If it's an analysis issue, mention the file type and audio duration...

**Comment by Mattk70** [comment]
Hi @David-Bevan, the application is not signed. To install an unsigned app, you need to hold the control key before opening the download from the downloads folder, as detailed in the instructions in the screenshot. See also: https://support.apple.com/en-gb/guide/mac-help/mh40616/mac 

It may also be that you do not have administrator privileges for the Mac in question. If that's the case, you will need an administrator to do this. I hope that helps!

**Comment by Mattk70** [comment]
This issue was due to the Mac in question having an Intel processor, which is not supported.
